web/src/components/PoisMobile.tsx

```diff
@@ -163,24 +191,18 @@ const PoisMobile = ({
             <StyledIcon icon={faArrowLeft} direction={direction} />
           </BackNavigation>
         )}
-        {/* To use geolocation in a development build you have to start the dev server with "yarn start --https" */}
-        <GeolocateControl
-          style={{
-            bottom: bottomActionSheetHeight,
-            position: 'absolute',
-            right: 0,
-          }}
-          positionOptions={{ enableHighAccuracy: true }}
-          trackUserLocation
-          position='bottom-right'
-        />
       </MapView>
       <BottomActionSheet
         title={!currentFeatureOnMap ? t('listTitle') : undefined}
         toolbar={toolbar}
         ref={sheetRef}
         setBottomActionSheetHeight={setBottomActionSheetHeight}
         direction={direction}>
+        <GeoLocateContainer
+          id='geolocate'
+          ref={geolocatePosition}
+          isLastSnapPoint={bottomActionSheetHeight >= getSnapPoints(height)[2]}
```

Comment:
```suggestion
          isLastSnapPoint={bottomActionSheetHeight > getSnapPoints(height)[2]}
```
I'm fine if we don't want to show the geocontrol on the last two snapPoints (as you implemented) but then the wording is misleading. So either rename the variable if the last two snapPoints are affected or change it to my suggestion